src/common/components/build-row/index.js

```diff
@@ -12,7 +12,7 @@ const BuildRow = (props) => {
     architecture,
     buildId,
     duration,
-    status,
+    colour,
```

Comment:
Even though users aren't linked through to this page any more, I decided it's worth fixing in case we make it acessibile in the future.